### PR TITLE
Add docs to readme

### PR DESCRIPTION
**Pull request by JaydenElliott**
Added docs badge to readme for easy access to docs (had to go to crates.io to get them previously).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
 [![Build Status][ci-badge]][ci-url]
 [![Crates.io][crates-badge]][crates-url]
+[![Docs][docs-badge]][docs-url]
 
 [ci-badge]: https://travis-ci.org/itarato/toxiproxy_rust.png?branch=master
 [ci-url]: https://travis-ci.org/itarato/toxiproxy_rust
 [crates-badge]: https://img.shields.io/crates/v/toxiproxy_rust.svg
 [crates-url]: https://crates.io/crates/toxiproxy_rust
+[docs-url]: https://docs.rs/toxiproxy_rust/0.1.6/toxiproxy_rust/
```
Comment (itarato) [inline README.md:11]:
```suggestion
[docs-url]: https://docs.rs/toxiproxy_rust/latest/toxiproxy_rust/
```

This way we wouldn't need to update the link on each version change.

**Review by itarato — COMMENTED** [review]
Sorry for the late notice. Much appreciated for the change, its indeed helpful. I added one minor comment to make a link non version dependent, the rest looks great. Thank you for this!

File: README.md (hunk 1 above)
Comment (itarato) [reply README.md:11]: I'll just forward fix this actually. Better to have this sooner.

**Review by itarato — APPROVED** [review]

**Comment by JaydenElliott** [comment]
Sorry I just saw this, thanks for updating it.

**Pull request merged**